src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -148,11 +147,7 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   const isScheduled = batchIdToDate(order.validFrom) > now
   const isActiveNextBatch = batchId === order.validFrom
   const isFirstActiveBatch = batchId === order.validFrom + 1 && msRemainingInBatch > 60 * 1000 // up until minute 4
-
-  const isUnlimited = useMemo(() => isOrderUnlimited(order.priceNumerator, order.priceDenominator), [
-    order.priceDenominator,
-    order.priceNumerator,
-  ])
+  const isUnlimited = order.isUnlimited
   const isActive = useMemo(() => order.remainingAmount.eq(order.priceDenominator), [
     order.priceDenominator,
```

Comment:
At this point should we put `isActive and isFilled` on the order itself in `decodeAuctionElements`?

Reply:
Yeah, probably. Will get there in a next PR.

This change is the minimum we need to do a higher level filtering of liquidity orders.